### PR TITLE
Added datatype support

### PR DESCRIPTION
**Pull request by splautz**
- Added support for some expanded data column types, specifically related to JSON columns.

Merging this will allow me to close my forked repo, as well as my published pypi package

### CODE REVIEW

File: alchemyjsonschema/__init__.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 import logging
 from collections import OrderedDict
 import sqlalchemy.types as t
+import sqlalchemy.dialects.postgresql as pgt
```
Comment (podhmo) [inline alchemyjsonschema/__init__.py:5]: It's OK? use it with the environment that postgresql is not installed.

```console
$ python -m venv /tmp/foo
$ . /tmp/foo/bin/activate
$ pip install sqalchemy
$ python
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sqlalchemy.dialects
>>> import sqlalchemy.dialects.postgresql
```

OK

File: setup.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 setup(
     name="alchemyjsonschema",
     version=open("VERSION").read().strip(),
-    description="mapping jsonschema for sqlalchemy models",
+    description="A package for generating json-schema models from sqlalchemy models.",
```
Comment (podhmo) [inline setup.py:35]: exactly.

**Comment by podhmo** [comment]
Is this a related issue? If so, create another PR for this.

- #36

**Comment by podhmo** [comment]
released as 0.8.0 🚀

File: alchemyjsonschema/__init__.py (hunk 1 above)
Comment (splautz) [reply alchemyjsonschema/__init__.py:5]: Yes, this previously worked fine as long as the connected database was not postgresql, but given postgresql is a very commonly used database I think the addition of the JSON types used by postgresql was useful.

**Pull request merged**